src/ios-deploy/ios-deploy.m

```diff
@@ -2058,7 +2056,8 @@ void handle_device(AMDeviceRef device) {
           CFIndex size = sizeof(keys)/sizeof(CFStringRef);
           options = CFDictionaryCreate(NULL, (const void **)&keys, (const void **)&values, size, &kCFTypeDictionaryKeyCallBacks, &kCFTypeDictionaryValueCallBacks);
 
-          connect_and_start_session(device);
+          // Incremental installs do not seem to need a session to be started with AMDeviceStartSession() and can instead
```

Comment:
I had to read this comment a couple times to understand what you were saying here. I'd simplify this comment to "Incremental installs should be done without a session started". If someone wants to know more they will look at the blame/git-history.

Reply:
Done.